src/screens/FlashCardsView/UI/FlashCardsViewCon.tsx

```diff
@@ -24,34 +27,143 @@ export const FlashCardsViewCon: FC<FlashCardsListConProps> = (props) => {
   const [flashcardName, setFlashcardName] = useState<string>(name);
   const [buttonDisable, setButtonDisable] = useState<boolean>(false);
   const [wordsData, setWordsData] = useState<WordDef[]>(words);
+  const [isAddOpen, setIsAddOpen] = useState<boolean>(false);
+  const [isEditOpen, setIsEditOpen] = useState<boolean>(false);
+  const [activeId, setActiveId] = useState<number | null>(null);
+  const [loading, setLoading] = useState<boolean>(false);
+  const [wordExamplePreview, setWordExamplePreview] = useState<string>('');
+  const [newExample, setNewExample] = useState<string>('');
   const [data, setData] = useRecoilState(FlashCardsDataState);
+  const item = wordsData.find((item) => item.id === activeId || null);
+  const [wordName, setWordName] = useState<string>(item?.name || '');
+  const [wordMean, setWordMean] = useState<string>(item?.mean || '');
+  const [wordLang, setWordLang] = useState<string>(item?.lang || '');
+  const [wordExample, setWordExample] = useState<string>(item?.example || '');
+  const apiKey = useRecoilValue(APIKeyState);
+  const [newWord, setNewWord] = useState<string>('');
+  const [newMean, setNewMean] = useState<string>('');
+  const [newLang, setNewLang] = useState<string>('');
+  const [addWordsData, setAddWordsData] = useState<WordDef[]>([]);
```

Comment:
これの原因になるのでこの行は無くしてください。
```
Warning: Encountered two children with the same key, `0`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
```